src/Resources/config/seo.xml

```diff
@@ -8,8 +8,8 @@
         <service
                 id="cmf_sonata_phpcr_admin_integration.seo.extension.metadata"
```

Comment:
i think each id has the the following design:

`cmf_sonata_phpcr_admin_integration.[bundleName].[specific]`

(hopefully)